.github/workflows/build.yaml

```
@@ -0,0 +1,27 @@
+  build_typst_documents:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Typst
+        uses: lvignoli/typst-action@main
+        with:
+          source_file: ryotaro_kimura.typ
+
+      - name: copy pdf files to build dir
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2035:info:2:4: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)